### PR TITLE
Feat(eos_cli_config_gen): Add 'router service-insertion' CLI

### PR DESCRIPTION
**Pull request by gmuloc**
## Change Summary

implements

```
router service-insertion
```

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes

Added in schem

## How to test

molecule

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-service-insertion.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!--
+  ~ Copyright (c) 2023 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+=== "Table"
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-service-insertion.md:6]: Remember to add this table to the input variables md file manually.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/routing.j2 [hunk 2]
```diff
@@ -38,6 +39,8 @@
 {%     include 'documentation/arp.j2' %}
 {## Router General #}
 {%     include 'documentation/router-general.j2' %}
+{## Router Service Insertionl #}
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/routing.j2]:
```suggestion
{## Router Service Insertion #}
```

Comment (gmuloc) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/routing.j2]: [9bcca08](https://github.com/aristanetworks/ansible-avd/pull/3220/commits/9bcca081daa178f07fc19cc3e153c968a3a61907)

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-service-insertion.md (hunk 1 above)
Comment (gmuloc) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-service-insertion.md:6]: [9bcca08](https://github.com/aristanetworks/ansible-avd/pull/3220/commits/9bcca081daa178f07fc19cc3e153c968a3a61907)

Added STUN as well which was missed in #3147

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_service_insertion.schema.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  router_service_insertion:
+    type: bool
+    description: Configure network services inserted to data forwarding
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_service_insertion.schema.yml]: This should be a dict with an enabled flag. There are already TOIs with commands underneath this context, so even though we don't need those today, we should not block for adding them later.

**Review by ClausHolbechArista — APPROVED** [review]

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Pull request merged**